src/paths/ForwardingNumber.ts

```diff
@@ -0,0 +1,64 @@
+// This is Generated Source.
+import PathSegment from "../PathSegment";
+import ForwardingNumberInfo from "../definitions/ForwardingNumberInfo";
+import PagingResult from "../PagingResult";
+
+export default class ForwardingNumber extends PathSegment {
+    constructor(prv: PathSegment, id?: string, service?) {
+        super("forwarding-number", id, prv, service);
+    }
+
+    /**
+     *  Add New Forwarding Number
+     */
+    post(body: PostBody): Promise<ForwardingNumberInfo> {
+        return this.getService().send({
+          body: body,
+          method: "post",
+          query: undefined,
+          url: this.getEndpoint(true),
+        }).then(function (res) {
+            return res.json();
+        });
```

Comment:
Then I'll keep returning json based on the swagger.json. I don't think I can move 'return json' to parent, because returning json or other type is based on the swagger.json, for Binary type the original response is returned, for delete methods nothing is returned, anyway what should be returned totally depends on the swagger.jon.